media/js/mozorg/try-picture-in-picture.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/* eslint no-unused-vars: [2, { "varsIgnorePattern": "onYouTubeIframeAPIReady" }] */
+
+// YouTube API hook has to be in global scope
+window.onYouTubeIframeAPIReady = function () {
+    'use strict';
+
+    // Play the video only once the API is ready.
+    Mozilla.pipVideoPlay();
+};
+
+(function () {
+    'use strict';
+
+    var videoLink = document.querySelector('.js-video-play');
+    var src = 'https://www.youtube.com/iframe_api';
+
+    function loadScript() {
+        var tag = document.createElement('script');
+        tag.src = src;
+        var firstScriptTag = document.getElementsByTagName('script')[0];
+        firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+    }
+
+    function isScriptLoaded() {
+        return document.querySelector('script[src="' + src + '"]')
+            ? true
+            : false;
+    }
+
+    function playVideo() {
+        var videoId = videoLink.getAttribute('data-id');
+        var title = videoLink.getAttribute('data-video-title');
+
+        // return early if youtube API fails to load or is blocked.
+        if (typeof window.YT === 'undefined') {
+            return;
+        }
+
+        new window.YT.Player(videoLink, {
+            width: 500,
+            height: 281,
+            videoId: videoId,
+            playerVars: {
+                start: 158,
```

Comment:
Although the chapter break at this timestamp is labeled "picture in picture" it actually drops in right at the end of a different topic and she starts talking about PiP several seconds later, so it's kind of weird. I think this should be `169`, which is right where she changes topics to start talking about PiP.